packages/connectivity/example/android/app/build.gradle

```diff
@@ -52,6 +52,9 @@ flutter {
 }
 
 dependencies {
+    implementation 'androidx.lifecycle:lifecycle-runtime:2.1.0'
```

Comment:
remove these deps, the engine POM makes sure these are pulled in (once https://github.com/flutter/flutter/pull/41885 lands)